sparkmagic/sparkmagic/magics/sparkmagicsbase.py

```diff
@@ -37,6 +39,29 @@ def __init__(self, shell, data=None, spark_events=None):
             spark_events = SparkEvents()
         spark_events.emit_library_loaded_event()
 
+    def execute_local(self, cell, input_variable_name, type, output_variable_name, session_name):
+        input_variable_value = self.shell.user_ns[input_variable_name]
+
+        if not input_variable_value:
+            raise BadUserDataException("Value of specified -o variable is None!")
+
+        if not output_variable_name:
+            output_variable_name = input_variable_name
+
+        if type == 'str':
```

Comment:
`type` is a reserved word; how about `var_type`?
Also, let's make the types constants and compare after lowercasing the user input?

Reply:
sure, good idea